sonic-pcied/scripts/pcied

```diff
@@ -157,6 +160,40 @@ class DaemonPcied(daemon_base.DaemonBase):
         if self.resultInfo is None:
             return
 
+        try:
+            stream = open(PLATFORM_PCIE_YAML_FILE, 'r')
+
+            # Load contents of the PCIe YAML file into a dictionary object
+            dictionary = yaml.safe_load(stream)
+            
+            # Iterate through each element in dict; extract the BDF information and device ID for that element
+            for elem in dictionary:
+                bus = int(elem.get('bus'), 16)
+                device = int(elem.get('dev'), 16)
+                func = int(elem.get('fn'), 16)
+                pcieDeviceID = elem.get('id')
+
+                # Form the PCI device address from the BDF information above
+                pcie_device = ("{:02x}:{:02x}.{:01d}".format(bus, device, func))
+                
+                # Form and run the command to read the device ID from the PCI device
+                cmd = ["setpci", "-s", pcie_device, "2.w"]
+                output = subprocess.check_output(cmd)
```

Comment:
@assrinivasan I meant the comment for this entire block of code `that you are reading the PCI device id`

Reply:
Added in latest commit